src/index.scss

```diff
@@ -172,6 +172,66 @@ $select-input-height: 40px; // Bootstrap input height
     color: #777;
   }
 }
+// COLORS ======================================================================
+
+.xo-color-running {
+  @extend .text-success;
+}
+
+.xo-color-halted {
+  @extend .text-danger;
+}
+
+.xo-color-suspended {
+  @extend .text-info;
+}
+
+.xo-color-unknown, .xo-color-paused {
+  @extend .text-muted;
+}
+
+.xo-color-pending {
+  @extend .text-warning;
+}
+
+// POWER STATES ================================================================
+
+.xo-icon-running {
```

Comment:
Should be in the icon section.
IMHO they should be prefixed by `xo-icon-vm-state-`.